examples/tutorials/analysis-1d/spectral_analysis_rad_max.py

```diff
@@ -41,15 +41,15 @@
 The directional cut is typically an angular distance from the assumed
 source position, :math:`\\theta`. The
 `gamma-astro-data-format <https://gamma-astro-data-formats.readthedocs.io/en/latest/>`__
-specifications offer two different ways to store this information: \* if
-the same :math:`\\theta` cut is applied at all energies and offsets, `a
-`RAD_MAX`
-keyword <https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/point_like/#rad-max>`__
-is added to the header of the data units containing IRF components. This
-should be used to define the size of the ON and OFF regions; \* in case
-an energy- (and offset-) dependent :math:`\\theta` cut is applied, its
-values are stored in additional `FITS` data unit, named
-``RAD_MAX_2D` <https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/point_like/#rad-max-2d>`__.
+specifications offer two different ways to store this information:
+
+* if the same :math:`\\theta` cut is applied at all energies and offsets, a
+  `RAD_MAX <https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/point_like/#rad-max>`__
+  keyword is added to the header of the data units containing IRF components. This
+  should be used to define the size of the ON and OFF regions;
+* in case an energy- (and offset-) dependent :math:`\\theta` cut is applied, its
```

Comment:
I would adjust this too:
`energy-dependent (and offset-dependent) `

Reply:
done